### PR TITLE
User profiles

### PR DESCRIPTION
**Pull request by jkelly8823**

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -2,6 +2,116 @@ class CoursesController < ApplicationController
   before_action :authenticate_user!
   before_action :set_course, only: %i[ show edit update destroy ]
 
+  def getStats  
+    if @target_course_id.is_a?(ActiveRecord::Base)
```
Comment (kunalvudathu) [inline app/controllers/courses_controller.rb:6]: is there code coverage for this? if not, can you add unit cases for this?

File: db/schema.rb [hunk 2]
```diff
@@ -109,6 +109,10 @@
     t.string "unconfirmed_email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "firstName"
+    t.string "lastName"
+    t.string "firstname", default: ""
+    t.string "lastname", default: ""
```
Comment (kunalvudathu) [inline db/schema.rb:127]: looks like there duplicates

**Review by kunalvudathu — COMMENTED** [review]
Looks great! overall!

File: db/schema.rb (hunk 2 above)
Comment (jkelly8823) [reply db/schema.rb:127]: made a new migration

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (jkelly8823) [reply app/controllers/courses_controller.rb:6]: Tests updated

**Review by kunalvudathu — APPROVED** [review]

**Review by kunalvudathu — APPROVED** [review]

**Pull request merged**